### PR TITLE
add frontend files

### PR DESCRIPTION
**Pull request by Gasta-Marc-Oliver**
Hello evryone! Please review my code. Thank you!

### CODE REVIEW

File: timer.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Timer</title>
+    <link rel="stylesheet" href="oras.css" />
```
Comment (alima-kailamarie) [inline timer.html]: Your css name file didn't match your uploaded css file here.

File: timer.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      <div class="section">
+        <h2>Timer</h2>
+        <button id="start">Start</button>
+        <button id="pause" disabled>Pause</button>
+        <button id="resume" disabled>Resume</button>
+        <button id="stop">Stop</button>
+        <p id="count">0</p>
+      </div>
+    </div>
+    <script src="time.js"></script>
```
Comment (alima-kailamarie) [inline timer.html]: Same with your css name file. Your js name file didn't match also your uploaded js file here.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+let intervalId;
+
+let startButton = document.getElementById("start");
+let pauseButton = document.getElementById("pause");
+let resumeButton = document.getElementById("resume");
+let stopButton = document.getElementById("stop");
+let countSpan = document.getElementById("count");
+
+startButton.addEventListener("click", function () {
+  intervalId = setInterval(function () {
```
Comment (alima-kailamarie) [inline script.js]: Refrain creating a function inside another function.
Refer to the coding guidelines below:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR0AgTGRhVpMnMrBg9rIM05xqWwT9L0CCuzzuA_Yri3u3BW5iuTvvRNhkN0_aem_AeZMm9DFw7FXEQYpH4WcqolKKh_OZYLYbQrP18QItx6hu_p7pGos7XXWnUIFlXcqBpk_A6mKXBQ_SAx2CynH7k15#slide=id.g2651733a4ec_0_9

File: script.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+});
+
+pauseButton.addEventListener("click", function () {
+  clearInterval(intervalId);
+  pauseButton.disabled = true;
+  resumeButton.disabled = false;
+});
+
+resumeButton.addEventListener("click", function () {
+  intervalId = setInterval(function () {
```
Comment (alima-kailamarie) [inline script.js]: Refrain creating a function inside another function.
Refer to the coding guidelines below:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR0AgTGRhVpMnMrBg9rIM05xqWwT9L0CCuzzuA_Yri3u3BW5iuTvvRNhkN0_aem_AeZMm9DFw7FXEQYpH4WcqolKKh_OZYLYbQrP18QItx6hu_p7pGos7XXWnUIFlXcqBpk_A6mKXBQ_SAx2CynH7k15#slide=id.g2651733a4ec_0_9

**Review by alima-kailamarie — CHANGES_REQUESTED** [review]
I added some reviews in your files.

**Comment by Gasta-Marc-Oliver** [comment]
Hello @alima-kailamarie  please review my code again. Thank  you.

**Review by alima-kailamarie — APPROVED** [review]
Good work! You may now merge it to the main branch.

**Pull request merged**